_data/companies/fullcontact.yml

```diff
@@ -1,7 +1,7 @@
 ---
 name: FullContact
-wfh: Encouraged
+wfh: Restricted
```

Comment:
Could you explain what restricted means?

Reply:
Great point; that makes no sense. I’ll revise.